packages/test/src/workflows/stack-tracer.ts

```diff
@@ -24,3 +25,18 @@ export async function stackTracer(): Promise<[string, string]> {
   const second = await queryOwnWf(wf.stackTraceQuery);
   return [first, second];
 }
+
+export async function enhancedStackTracer(): Promise<[EnhancedStackTrace, EnhancedStackTrace]> {
```

Comment:
This is a duplicate of the workflow above, you can add a parameter there instead to control which query to issue instead of duplicating.